go.mod

```diff
@@ -28,7 +28,7 @@ require (
 	github.com/spf13/cobra v1.1.0
 	github.com/spf13/viper v1.7.1
 	github.com/stretchr/testify v1.6.1
-	github.com/tendermint/tendermint v0.34.0-rc4
+	github.com/tendermint/tendermint v0.34.0-rc5
```

Comment:
This is a bit odd: light/rpc/client-test.go imports "github.com/cosmos/iavl", which depends on "github.com/tendermint/tendermint"